src/main/java/io/vavr/control/Validation.java

```diff
@@ -455,6 +585,12 @@ public final T getOrElseGet(Function<? super E, ? extends T> other) {
 
     /**
      * Converts this Validation to an {@link Either}.
+     * <pre>{@code
+     * //following code converts validations of valid type to Either right type.
+     * Validation validation =  Validation.valid(1)
+     * Either either = validation.toEither()
+     * //otherwise if the validation contains invalid then it returns Either left type.
+     * * }</pre>
```

Comment:
There is one `*` too much here

Reply:
Hi Daniel,
I've addressed the review comments. My apologies for the initial commit which was having lot of mistakes, if something is still not good please let me know.
Thanks..

Reply:
Hi, sorry, I‘ve missed your comment. Thanks for all your effort - it looks nice now! :)